### PR TITLE
Quality of life improvements

### PR DESCRIPTION
**Pull request by scohen**
Just a couple quality of life improvements. 

We needed to increase the max connections beyond Ranch's 1024, which proved impossible because you can't specify transport_opts easily.

Also fixed a 1.6 compile due to the ordering of catch and rescue.

### CODE REVIEW

File: lib/thrift/generator/binary/framed/server.ex [hunk 1]
```diff
@@ -99,15 +99,15 @@ defmodule Thrift.Generator.Binary.Framed.Server do
...
-      rescue
-        unquote(rescue_blocks)
       catch kind, reason ->
         formatted_exception = Exception.format(kind, reason, System.stacktrace)
         Logger.error("Exception not defined in thrift spec was thrown: #{formatted_exception}")
         error = Thrift.TApplicationException.exception(
           type: :internal_error,
           message: "Server error: #{formatted_exception}")
         {:server_error, error}
+      rescue
```
Comment (nichochar) [inline lib/thrift/generator/binary/framed/server.ex]: Out of curiosity, should `rescue` always come after `catch` ? If so maybe we can add some static analysis and catch this at the erbert level

Comment (thecodeboss) [reply lib/thrift/generator/binary/framed/server.ex]: Elixir 1.6 has a warning built in for this I believe

Comment (scohen) [reply lib/thrift/generator/binary/framed/server.ex]: @nichochar, yes, rescue should always come after catch (at least according to the new style rules). I don't think there's any technical reason for this, but 1.6 now emits a warning when this isn't the case.

**Comment by scohen** [comment]
Does anyone know why the test failed?

**Comment by jparise** [comment]
@scohen, I don't see the original test failure, but it looks green now. Did you trigger a rebuild?

There are still some flaky service tests in the suite due to the way that the Erlang Thrift server non-deterministically closes its sockets and shuts down its processes.

**Comment by scohen** [comment]
That's... odd... I see it as green as well. 
Going to merge.

**Comment by scohen** [comment]
The tests that failed were SSL related, so it might have been the erlang quirkiness you mentioned.

**Pull request merged**